sql/service_wsrep.cc

```diff
@@ -167,7 +167,7 @@ extern "C" void wsrep_handle_SR_rollback(void *bf_thd_ptr,
                                          void *victim_thd_ptr)
 {
   DBUG_ASSERT(victim_thd_ptr);
-  if (!victim_thd_ptr) return;
+  if (!victim_thd_ptr || (bf_thd_ptr && !wsrep_on(bf_thd_ptr))) return;
```

Comment:
I think bf_thd_ptr check is not needed , 
extern "C" my_bool wsrep_on(const void *thd)
{
  return (int)(WSREP(((const THD*)thd)));
}

#define WSREP(thd) \
  (thd && WSREP_NNULL(thd))

Reply:
yes, good observation, I will remove the bf_thd_ptr check from this condition

Reply:
A fixup commit has been pushed that addresses this issue